scripts/core/device.yml

```
@@ -395,10 +393,10 @@ params:
       name: hDevice
       desc: |
             [in] handle of the device to partition.
-    - type: "$x_device_partition_property_value_t*"
+    - type: const $x_device_partition_property_t*
       name: Properties
       desc: |
-            [in] null-terminated array of <property, value> pair of the requested partitioning.
+            [in] null-terminated array of <property, value> pairs.
```

I think this description needs to mention what enumeration is expected to be used, otherwise there would be no relation between `ur_device_partition_t` and this list of properties.

```suggestion
            [in] null-terminated array of <$_device_partition_t enum, value> pairs.
```